app/package.json

```diff
@@ -29,6 +29,7 @@
     "cors": "^2.8.5",
     "d3-array": "^2.7.1",
     "d3-axis": "^2.0.0",
+    "d3-brush": "^2.1.0",
```

Comment:
Sidenote: I think we need to make sure that we're using individual d3 package versions that belong together. Not all the types are up-to-date as well … I tried before but failed because @types/d3 wasn't at v6 back then, but it should be now.